src/components/dls/QuranWord/MadaniWordText.tsx

```diff
@@ -1,25 +1,40 @@
 import React from 'react';
 import styled from 'styled-components';
 import { decamelizeKeys } from 'humps';
+import { CharType } from 'types/WordType';
 
 type MadaniWordTextProps = {
   text: string;
-  fontVersion: string;
+  font: string;
+  charType: string;
 };
 
+const DEFAULT_FONT_FAMILY = 'UthmanicHafs1Ver17';
+
 const UTHMANI_HAFS_FONTS = decamelizeKeys({
-  qpcUthmaniHafs: 'UthmanicHafs1Ver17',
-  textUthmani: 'Madani',
+  qpcUthmaniHafs: DEFAULT_FONT_FAMILY,
+  textUthmani: 'meQuran',
+  textIndopak: 'IndoPak',
```

Comment:
nit: If we are having all the word types be in this component, it might be worthwhile to call it something other than "MadaniWordText". The name is misleading